From: Richard B Winters <rik@mmogp.com>
Date: Sun, 3 May 2015 16:24:18 -0400
Subject: Fix pcsd.conf destination in pcsd makefile

 - The default install destination for pcsd.conf is /etc/
   sysconfig/pcsd, this needs changing on debian

Change-Id: I49610e431a37da21e1b350165c98626eae1fe5c6
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 0bc1767..6cf519b 100644
--- a/Makefile
+++ b/Makefile
@@ -39,7 +39,7 @@ install_pcsd:
 	mkdir -p ${DESTDIR}/var/log/pcsd
 	mkdir -p ${DESTDIR}${PREFIX}/lib/
 	cp -r pcsd ${DESTDIR}${PREFIX}/lib/
-	install -m 644 -D pcsd/pcsd.conf ${DESTDIR}/etc/sysconfig/pcsd
+	install -m 644 -D pcsd/pcsd.conf ${DESTDIR}/etc/default/pcsd.conf
 	install -d ${DESTDIR}/etc/pam.d
 	install  pcsd/pcsd.pam ${DESTDIR}/etc/pam.d/pcsd
 	install -m 700 -d ${DESTDIR}/var/lib/pcsd
